WMS5541 - Updated to latest GUI and LL standards

rpastijn opened the pull request:
# WMS 5541

# General requirements
1. Information in the workshop is adequate and updated -> Yes
2. Code is correct and working -> Yes
3. Links are correct -> Yes
4. Make sure you enter the help email link in your manifest.json -> Yes
5. Please make sure WMS URLs are updated as needed after your PR is approved -> Ok


# Checklist - Refer to the QA document for the complete list 
Please confirm that the following is completed before submitting your PR

- [X ] All filenames are lower case (including folders, images, files, etc.)
- [X] Filenames are descriptive
- [X] Your workshop folder structure should be similar to the one used in the sample workshop (https://github.com/oracle-livelabs/common/tree/main/sample-livelabs-templates/sample-workshop)
- [X] Are you using multiple versions (desktop/, sandbox/, tenancy/)? Make sure that each of them contains a manifest.json and an index.html
- [X] Image references in markdown contain an alternate text

arabellayao reviewed (changes requested):
Hi! Please fix the following issues of your PR:
1. Only workshop team members can update the workshop, but your name is not listed as the workshop team for this workshop. Can you ask Connor Mcdonald or David Start (who are listed) to add you as the workshop team?
2. All images should have alternative texts in the Markdown files for accessibility
Please refer to the [self QA form](https://objectstorage.us-ashburn-1.oraclecloud.com/p/MKKRgodQ0WIIgL_R3QCgCRWCg30g22bXgxCdMk3YeKClB1238ZJXdau_Jsri0nzP/n/c4u04/b/qa-form/o/QA.docx) for details and let me know if you have any questions. Thanks!

rpastijn commented:
Updated the location for the alternative text (put it between quotes, now moved it to the brackets per Q&A). I have asked Connor to add me to the team. He is very busy for OCW so I asked David. Just to be sure, I have email trail where Connor thanks me for changing this, forwarded this to David.

rpastijn commented:
Connor just added me as support contact in WMS. Please let me know if this is not enough.
